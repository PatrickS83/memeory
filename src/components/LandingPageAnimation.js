import React from 'react';

const LandingPageAnimation = () => (
  <div id="splash">
    <div className="anim">
      <div id="loader">
        <img src="./img/memeory-logo-white.png" id="landingLogo" alt="Landingpage Logo" />
      </div>
    </div>
  </div>
);

export default LandingPageAnimation;
