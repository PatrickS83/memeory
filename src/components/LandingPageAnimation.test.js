import React from 'react';
import { shallow } from 'enzyme';
import LandingPageAnimation from './LandingPageAnimation';

describe('<LandingPageAnimation/>', () => {
  it('renders without crashing', () => {
    const wrapper = shallow(<LandingPageAnimation />);
    expect(wrapper.length).toEqual(1);
  });

  it('renders the LandingPageAnimation as saved in snapshot', () => {
    const wrapper = shallow(<LandingPageAnimation />);
    expect(wrapper).toMatchSnapshot();
  });
});
