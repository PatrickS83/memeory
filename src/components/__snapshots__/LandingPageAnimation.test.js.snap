// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<LandingPageAnimation/> renders the LandingPageAnimation as saved in snapshot 1`] = `
<div
  id="splash"
>
  <div
    className="anim"
  >
    <div
      id="loader"
    >
      <img
        alt="Landingpage Logo"
        id="landingLogo"
        src="./img/memeory-logo-white.png"
      />
    </div>
  </div>
</div>
`;
